docs/integrations/sources/file.md

```diff
@@ -31,17 +31,17 @@ This page contains the setup guide and reference information for the Files sourc
 Set this to active if you want to add the [User-Agent header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent) to requests (inactive by default).
 
 #### GCS: Google Cloud Storage
-- `Service Account JSON` (Required for **private** buckets) 
```

Comment:
Looks like my IDE did auto-cleanup on markdown here. 

You can use the ⚙️ icon in the PR diff view to suppress whitespace changes, if helpful.

![image](https://github.com/airbytehq/airbyte/assets/18150651/f5d3733f-9c93-46e7-9908-bb0a19ec829d)